packages/core/src/xr/feature/camera/XRCameraManager.ts

```diff
@@ -0,0 +1,126 @@
+import { Matrix } from "@galacean/engine-math";
+import { Camera } from "../../../Camera";
+import { Engine } from "../../../Engine";
+import { CameraClearFlags } from "../../../enums/CameraClearFlags";
+import { XRCamera } from "../../input/XRCamera";
+import { XRTrackedInputDevice } from "../../input/XRTrackedInputDevice";
+
+/**
+ * The manager of XR camera.
+ */
+export class XRCameraManager {
+  /**
+   * Return fixed foveation of the camera.
+   */
+  get fixedFoveation(): number {
+    const { _platformSession: platformSession } = this._engine.xrManager.sessionManager;
+    if (platformSession) {
+      return platformSession.fixedFoveation;
+    } else {
+      throw new Error("XR session is not available.");
+    }
+  }
+
+  set fixedFoveation(value: number) {
+    const { _platformSession: platformSession } = this._engine.xrManager.sessionManager;
+    if (platformSession) {
+      platformSession.fixedFoveation = value;
+    } else {
+      throw new Error("XR session is not available.");
+    }
+  }
+
+  /**
+   * @internal
+   */
+  constructor(private _engine: Engine) {}
+
+  /**
+   * Attach the camera to the specified input type(Camera, LeftCamera or RightCamera).
+   * The camera entity need to be moved to the XROrigin entity.
+   * @param type - The input type
+   * @param camera - The camera to be attached
+   */
+  attachCamera(
+    type: XRTrackedInputDevice.Camera | XRTrackedInputDevice.LeftCamera | XRTrackedInputDevice.RightCamera,
+    camera: Camera
+  ): void {
+    this._engine.xrManager.inputManager.getTrackedDevice<XRCamera>(type).camera = camera;
+  }
+
+  /**
+   * Detach the camera from the specified input type.
+   * @param type - The input type
+   * @returns The camera that was detached
+   */
+  detachCamera(
+    type: XRTrackedInputDevice.Camera | XRTrackedInputDevice.LeftCamera | XRTrackedInputDevice.RightCamera
+  ): Camera {
+    const xrCamera = this._engine.xrManager.inputManager.getTrackedDevice<XRCamera>(type);
+    const preCamera = xrCamera.camera;
+    xrCamera.camera = null;
+    return preCamera;
+  }
+
+  /**
+   * Get the camera by the specified input type.
+   * @param type - The input type
+   * @returns The camera
+   */
+  getCameraByType(
```

Comment:
- There are two entrances to configure this function, which is a bit strange.
- Maybe we can try monocular and binocular cameras